test/OpenTelemetry.Instrumentation.AspNetCore.Tests/BasicTests.cs

```diff
@@ -1046,115 +1032,45 @@ void ConfigureTestServices(IServiceCollection services)
                 })
                 .Build();
 
-            services.AddOpenTelemetry()
-                .WithTracing(builder => builder
-                    .AddInstrumentation(() => this.tracerProvider));
-        }
+        var builder = WebApplication.CreateBuilder();
+        builder.Logging.ClearProviders();
+        var app = builder.Build();
 
-        using (var client = this.factory
```

Comment:
@ngruson Why was this change reverted? This one works fine right?

Reply:
Yes, over the weekend I was struggling to get that one to work properly. 
I just fixed it and now it is working with the use of `WebApplicationFactory'.
Except this particular test is skipped during CI, but maybe that's something for another PR.

Reply:
you could remove the skip in this PR itself. Also, could you revert the changes in .sln file, unintentional changes i guess.